utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -313,6 +313,11 @@ WBWIIterator* WriteBatchWithIndex::NewIterator(
                               &(rep->comparator));
 }
 
+WBWIIterator* WriteBatchWithIndex::NewIterator(uint32_t cf_id) const {
+  return new WBWIIteratorImpl(cf_id, &(rep->skip_list), &rep->write_batch,
```

Comment:
nit: Sanity check `cf_id` is legit. The internal usage should be able to always get a legit cf_id. But this API seems to be a public API that users can use too.

Reply:
Moved to private.